medical/models/medical_patient.py

```diff
@@ -60,110 +34,79 @@ class MedicalPatient(orm.Model):
     form around medical.
     '''
     _name = 'medical.patient'
+    _description = 'Medical Patient'
     _inherits = {'res.partner': 'partner_id', }
 
-    def _get_default_patient_id(self, cr, uid, context=None):
+    @api.model
+    def _get_default_patient_id(self):
         """ Gives default patient_id """
-        patient_ids = self.search(cr, uid, [('name', '=', 'Libre')],
-                                  context=context)
-        if not patient_ids:
-            raise orm.except_orm(_('Error!'), _('No default patient defined'))
-
-        return patient_ids[0]
-
-    def _get_age(self, cr, uid, ids, field_name, arg, context=None):
-        res = {}
-        age = ''
+        domain = [('name', '=', 'Libre')]
```

Comment:
I just ported it as well but to be honest this concept of default patient seems strange for me: It is only used into `medical_appointment`  and I am not sure it was the right way to do it. Anyway I didn't modify it to avoid change into other modules. What is the best to do?